### PR TITLE
Create French Translation

### PR DESCRIPTION
**Pull request by Pandz18**
Hello , I have translated the contributing , README , index , details and the en.js files into French . Hope it is good.

### CODE REVIEW

**Comment by Pandz18** [comment]
> Deploy request for _hacktoberfest-checker_ pending review.
> 
> Review with commit [49b37f7](https://github.com/lukeocodes/hacktoberfest-checker/commit/49b37f7d851ef2099b0550706bfcb7efa274202d)
> 
> https://app.netlify.com/sites/hacktoberfest-checker/deploys

@lukeocodes I do not understand this error , did I do something wrong ?

File: .github/lang/CONTRIBUTING.Fr.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+Si vous souhaitez contribuer, faites d'abord quelques recherches [Code de conduite](./CODE_OF_CONDUCT.md).
+
+ ## Translations
+
+Les traductions sont disponibles en ** 3 ** parties, que vous pouvez soumettre en une seule demande d'extraction.
+
+ ### Documentation du site Web
+
+ Les fichiers de documentation du site Web existent dans le répertoire `. / Github / lang`. Si vous souhaitez soumettre une traduction, veuillez inclure les fichiers suivants avec le [code ISO à deux lettres pour votre langue] (https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes).
```
Comment (lukeocodes) [inline .github/lang/CONTRIBUTING.Fr.md:11]:
```suggestion
 Les fichiers de documentation du site Web existent dans le répertoire `./Github/lang`. Si vous souhaitez soumettre une traduction, veuillez inclure les fichiers suivants avec le [code ISO à deux lettres pour votre langue](https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes).
```

File: .github/lang/CONTRIBUTING.Fr.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+ z.B. chinois
+
+ - `.github/lang/README.zh.md`
+ - `.github/lang/CONTRIBUTING.zh.md`
+
+ Mettez à jour les liens entre vos nouveaux fichiers `README.zh.md` et` CONTRIBUTING.zh.md` afin que cliquer sur le lien *** Contributor *** dans` README.zh.md` mène directement à` CONTRIBUTING .zh .md` conduit.
```
Comment (lukeocodes) [inline .github/lang/CONTRIBUTING.Fr.md:18]:
```suggestion
Mettez à jour les liens entre vos nouveaux fichiers `README.zh.md` et `CONTRIBUTING.zh.md` afin que cliquer sur le lien ***Contributor*** dans `README.zh.md` mène directement à `CONTRIBUTING.zh.md` conduit.
```

File: .github/lang/CONTRIBUTING.Fr.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
+# Contribuant
+
+Si vous souhaitez contribuer, faites d'abord quelques recherches [Code de conduite](./CODE_OF_CONDUCT.md).
+
+ ## Translations
+
+Les traductions sont disponibles en ** 3 ** parties, que vous pouvez soumettre en une seule demande d'extraction.
```
Comment (lukeocodes) [inline .github/lang/CONTRIBUTING.Fr.md:7]:
```suggestion
Les traductions sont disponibles en **3** parties, que vous pouvez soumettre en une seule demande d'extraction.
```

File: .github/lang/CONTRIBUTING.Fr.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
+# Contribuant
+
+Si vous souhaitez contribuer, faites d'abord quelques recherches [Code de conduite](./CODE_OF_CONDUCT.md).
+
+ ## Translations
```
Comment (lukeocodes) [inline .github/lang/CONTRIBUTING.Fr.md:5]:
```suggestion
## Translations
```

**Comment by Pandz18** [comment]
@lukeocodes hi , I've crossed checked my translation files with the others to make sure , I haven't changed anything that would lead to this but both files match up . I'm confused

File: lang/Fr.js [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+    },
+
+    navigation: {
+      home: 'Accueil',
+      details: 'Détails',
+      check: 'Vérifier un référentiel',
+    },
+
+    check_repository: {
+      title: 'Est-ce qu'ils <strong>Hacktoberfest?</strong>',
```
Comment (lukeocodes) [inline lang/Fr.js:56]:
```suggestion
      title: 'Est-ce qu\'ils <strong>Hacktoberfest?</strong>',
```

**Review by lukeocodes — CHANGES_REQUESTED** [review]
Hey @Pandz18,

If you're still working on it, please make it a draft-PR until it is ready for review :) 

There are some issues across all the markdown. It all seems completely broken. See my first few comments (I won't go through the whole doc now).

There is also an escaping issue that has completely broken the `lang/Fr.js` file. Please test your translation.

image

Please also use lowercase `fr` for the file and directory names.

**Comment by lukeocodes** [comment]
@Pandz18 please don't worry about "Deploy request for `hacktoberfest-checker` pending review.". Once I am happy with the files, this allows me to make a preview build of the site so we can view your changes live before effecting the production site.

**Comment by Pandz18** [comment]
@lukeocodes I've gone through all the files again, I hope it's fixed now . Also how do I "test " my translations or create a draft PR?

**Comment by lukeocodes** [comment]
@all-contributors add @Pandz18 for translation, docs

**Pull request merged**